setup.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+import os
+import subprocess
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("command", choices=["install", "develop"])
+    parser.add_argument("--tiledb", type=str, required=False)
+    parser.add_argument("--debug", action="store_true")
+    parser.add_argument("-v", action="store_true")
+    args = parser.parse_args()
+
+    os.getcwd()
+
+    cmd = [
+        "pip",
+        "install",
+    ]
+
+    if args.command == "develop":
+        cmd.append("-e")
+
+    cmd.append(os.getcwd())
+
+    if args.tiledb:
+        cmd.append(f"-Cskbuild.cmake.define.TILEDB_PATH={args.tiledb}")
+
+    if args.debug:
+        cmd.append(f"-Cskbuild.cmake.define.CMAKE_BUILD_TYPE=DEBUG")
```

Comment:
I think that we should not use define here but instead:
```
-Cskbuild.cmake.build-type=Debug
```
https://scikit-build-core.readthedocs.io/en/latest/configuration.html#configuring-cmake-arguments-and-defines

Reply:
I changed this one